File: help/pt_pt/exercicios_da_ficha_3/Ficha03_008/Ficha03_008.java
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha03_008 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Valor de X\t: " );
        double x = Double.valueOf(keyb.nextLine());
        double logX = Math.log10( x ) ;
        System.out.print("\nLogaritmo 10 \t= " + logX );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 05 20:27

File: help/pt_pt/exercicios_da_ficha_3/Ficha03_008/Ficha03_008.js
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("Valor de X\t: " );// print text
    x = parseFloat(prompt("Valor de X\t: " ,"0.0") );
    
    document.write(x + "\n");// print input 
    var logX = 1/Math.log(10)*Math.log( x ) ;
    document.write("\nLogaritmo 10 \t= " + logX );
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 14:30
